### ISSUE TITLE
wrong redirecting on login

### ISSUE THREAD

**Issue by mahmoud-adel-mahmoud** [SEGUC17/Foobar#19]
1: Severity: ( low )

2:Reported: by Mahmoud

3:Description: When hitting on forget the password, and re-hit the login button it does not redirect to login page. Instead we have to refresh the page in order to get the login page again.

4: Steps to reproduce the issue: Press on enter. Then press on forget password. Then exit this form, Then try to enter again. You are redirected to reset password not the login page

5: Expected result: Correct redirect to login page after pressing on forget the password

**Comment by markarsanious** [comment]
yeah there seems to be a problem with the refresh but it is a "wrong redirection on login" after all so i think you used the wrong title to describe it
